api/reference/providers.adoc

```diff
@@ -226,7 +227,7 @@ Compound credentials set:
 [[specifying-connection-configurations]]
 === Specifying Connection Configurations
 When creating or updating providers, connection configurations can be set. Connection configurations can be used
-to specify resources such as an amqp event provider for Openstack or for adding Hawkular metrics.
+to specify resources such as an amqp event provider for Openstack, adding Hawkular metrics, or an Authentication token.
 
 Hawkular metrics:
 
```

Comment:
Is it possible an Openstack Cloud Manager also link to an existing Openstack Infra Provider?